data/human/news.txt

```diff
@@ -1829,8 +1829,117 @@ news "tourist pa system"
 			"Dear travelers: please take the time to adjust to normal gravity."
 		word
 			`"`
+phrase "pirate worlds"
+	word
+		"Albatross"
+		"Bloodsea"
+		"Buccaneer Bay"
+		"Covert"
+		"Deadman's Cove"
+		"Freedom"
+		"Greenrock"
+		"Haven"
+		"New Tortuga"
+		"Smuggler's Den"
+		"Stormhold"
+		"Thule"
+		"Zenith"
+
+phrase "rnas travel advisory [1]"
+	word
+		"Please be advised: a travel advisory remains in effect for"
+		"The Republic Navy strongly discourages travel to"
+	word
+		" "
+	word
+		"${pirate worlds}"
+	word
+		"."
+
+phrase "rnas travel advisory [2]"
+	word
+		"The Republic Navy Civilian Safety Board considers travel to"
+	word
+		" "
+	word
+		"${pirate worlds}"
+	word
+		" "
+	word
+		"to represent an unacceptable risk."
+		"to be extremely dangerous."
+		"to place Navy rescue personnel in unnecessary danger."
+
+phrase "rnas travel advisory [3]"
+	word
+		"Navigational hazard: "
```

Comment:
Does it really involve navigation? I would say it's more like a pirate hazard.

Reply:
I was trying to mimic the slightly stilted manner in which government alerts are often written. Happy to change it though if it seems odd.